uint8 robot_id
uint8 robot_level
uint16 current_HP
uint16 maximum_HP
uint16 shooter_barrel_cooling_value
uint16 shooter_barrel_heat_limit
uint16 chassis_power_limit